ui/src/sampleApp/studiesList.tsx

```diff
@@ -151,7 +146,14 @@ export function StudiesList() {
             />
           )}
           <Button
-            onClick={showNewStudyDialog}
+            onClick={() =>
+              showNewStudyDialog({
+                title: "New study",
+                textLabel: "Study name",
+                buttonLabel: "Create",
+                onConfirm: onCreateNewStudy,
```

Comment:
Not for this PR, but would it be possible to open a new study automatically once you create it, instead of bringing you back to the study list page?

Reply:
Yes. I haven't paid much attention to these pages because they won't be part of SD or AoU but that would be a nice, simple improvement.